db/schema.rb

```diff
@@ -120,6 +120,27 @@
     t.index ["sluggable_type", "sluggable_id"], name: "index_friendly_id_slugs_on_sluggable_type_and_sluggable_id"
   end
 
+  create_table "git_pull_request_reviews", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.bigint "git_pull_request_id", null: false
+    t.string "node_id", null: false
+    t.string "reviewer", null: false
```

Comment:
```suggestion
    t.string "reviewer", null: false
```

Can we be more specific here pls. `reviewer_github_handle`?